docs-drafts/ci-cd.md

```diff
@@ -0,0 +1,67 @@
+# Commonly Used CI/CD Tools
+
+|  Tool |  Purpose | Support  | 
+|---|---|---|
+|  *Argo CD* |  GitOps continuous delivery tool for Kubernetes |  Availabe within OpenShift |
```

Comment:
For the tools section specifically, does it need italics?

Reply:
I wanted to provide increase the visibility of the items in the Tool column, but it's not strictly necessary, but maybe we could use **bold** instead if that's preferable?